### PR TITLE
Extract actions on deletereleasebyid from models to release service

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: services/release/release.go [hunk 1]
```diff
@@ -130,3 +131,58 @@ func UpdateRelease(doer *models.User, gitRepo *git.Repository, rel *models.Relea
...
+	}
+
+	repo, err := models.GetRepositoryByID(rel.RepoID)
+	if err != nil {
+		return fmt.Errorf("GetRepositoryByID: %v", err)
+	}
+
+	if delTag {
+		_, stderr, err := process.GetManager().ExecDir(-1, repo.RepoPath(),
+			fmt.Sprintf("DeleteReleaseByID (git tag -d): %d", rel.ID),
```
Comment (lafriks) [inline services/release/release.go:152]: ~~This line is not needed~~

**Review by lafriks — CHANGES_REQUESTED** [review]

File: services/release/release.go (hunk 1 above)
Comment (lunny) [reply services/release/release.go:152]: Why do you think of that?

Comment (lafriks) [reply services/release/release.go]: It does not do anything as `fmt.Sprintf` is returning formatted string and it does not gets assigned to any variable

Comment (lafriks) [reply services/release/release.go:152]: oh sorry :D

**Review by lafriks — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work

**Pull request merged**